### PR TITLE
Author email article information section

### PR DESCRIPTION
**Pull request by discodavey**

### CODE REVIEW

File: src/components/molecules/author-information-list/author-information-list.scss [hunk 1]
```diff
@@ -19,6 +19,16 @@
   @include mixins.margin(36, "bottom");
 }
 
+.author-list__email {
+  @include mixins.margin(24, "bottom");
+  @include typography.body();
+
+  .author-list__email--heading {
+    display: inline;
+    font-size: 16px;
```
Comment (nlisgo) [inline src/components/molecules/author-information-list/author-information-list.scss:28]: should some of these values correspond to variants?

File: src/components/atoms/authors/authors.test.tsx [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
 import { Authors } from './authors';
 
 describe('authors', () => {
   it('should render correctly a list of authors', () => {
-    render(<Authors authors={[authors[0]]}/>);
-    const captainAmerica = screen.getByText('Steve Rogers');
+    const { container } = render(<Authors authors={[authors[0]]}/>);
 
-    expect(captainAmerica).toBeInTheDocument();
+    expect(container.querySelector('.authors-link')!.textContent).toStrictEqual('Steve Rogers');
```
Comment (nlisgo) [inline src/components/atoms/authors/authors.test.tsx:10]: This was necessary to protect against the introduce of whitespace before or after the author name.

See: https://github.com/elifesciences/enhanced-preprints-client/actions/runs/4377362358/jobs/7660739277

File: src/components/molecules/author-information-list/author-information-list.test.tsx [hunk 3]
```diff
@@ -107,4 +108,10 @@ describe('AuthorInformationList', () => {
     expect(screen.getByText('Peter Parker').nextSibling).toHaveTextContent('0000-0002-1234-5688');
     expect(screen.getByText('Valkyrie Brunnhilde').nextSibling?.nextSibling).not.toBeInTheDocument();
   });
+
+  it.each(authors.map((author) => createAuthorId(author)))('should contain an id with the author id', (id) => {
```
Comment (nlisgo) [inline src/components/molecules/author-information-list/author-information-list.test.tsx:112]: Is this not an instance where we would want to mock the function?

File: src/components/atoms/authors/authors.test.tsx [hunk 4]
```diff
@@ -31,6 +31,12 @@ describe('authors', () => {
     expect(expansionElement).not.toBeInTheDocument();
   });
 
+  it.each(authors.map((author) => createAuthorId(author)))('should contain a link with the author id', (id) => {
```
Comment (nlisgo) [inline src/components/atoms/authors/authors.test.tsx:34]: Feels like we should be mocking the `createAuthorId` function.

**Review by nlisgo — COMMENTED** [review]
Nothing blocking for me but would like an opinion on the potential to mock `createAuthorId` in `authors.test.tsx` and `author-information-list.test.tsx`

File: src/components/atoms/authors/authors.test.tsx (hunk 4 above)
Comment (discodavey) [reply src/components/atoms/authors/authors.test.tsx:34]: @will-byrne Your thoughts on @nlisgo comments on tests?

Comment (will-byrne) [reply src/components/atoms/authors/authors.test.tsx:34]: it does make the tests dependant on the `createAuthorId` working as expected, as this isnt an integration test mocking it is a good idea.

**Comment by discodavey** [comment]
@nlisgo Will did some changes to the tests. Can you look and confirm you are ok and if everyone is happy then this can go live this morning.

Chris has seen the changes and approved.

**Comment by discodavey** [comment]
@nlisgo We were hoping to get this live today. Will you be able to do that when you approve it this afternoon then?

**Review by nlisgo — APPROVED** [review]

**Pull request merged**